test/web.js

```diff
@@ -73,28 +73,30 @@ lab.experiment("OAuth", function() {
       };
 
       s.inject(request, function(response) {
-        Code.expect(response.statusCode).to.equal(500);
+        Code.expect(response.statusCode).to.equal(400);
+        Code.expect(response.result.message).to.equal("Password not strong enough.");
         ls.stop(done);
       });
     });
   });
 
-  lab.test("POST Create User (login API failure)", function(done) {
+  lab.test("POST Create User with invalid email", function(done) {
     ls.start(function(error) {
       Code.expect(error).to.be.undefined();
       var request = {
         method: "POST",
         url: "/create-user",
         payload: {
-          email: "webmaker@example.com",
+          email: "webmaker",
           username: "notgonnawork",
-          password: "CantGuessThis",
+          password: "CantGuessThis1234",
```

Comment:
keep this valid password, fix the email param and title back to what they were, and then undo the changes to the assertions.